jenkins-scripts/dsl/_configs_/OSRFBrewCompilationAnyGitHub.groovy

```diff
@@ -0,0 +1,23 @@
+package _configs_
+
+import javaposse.jobdsl.dsl.Job
+
+/*
+  -> OSRFBrewCompilation
+  -> GenericAnyJob
+
+  Implements:
+    - compiler warning
+*/
+class OSRFBrewCompilationAnyGitHub
+{
+  static void create(Job job,
+                     String github_repo,
+                     boolean enable_testing  = true,
+                     ArrayList supported_ros_distros = [])
```

Comment:
Is `supported_ros_distros` used here?

Reply:
oh I miss to add it to the GenericAnyJobGitHub call. My feeling is that the ROS support should be our of that generic class but that will need to wait a new refactor, not in this PR.

Reply:
I don't think it's used yet, but maybe someday. For now, it's just a pass-through.